### ISSUE TITLE
Inherit IDisposable in IFsiEvaluator

### ISSUE THREAD

**Issue by matthid** [tpetricek/FSharp.Formatting#341]
Continuation of https://github.com/matthid/Yaaf.FSharp.Scripting/issues/5.

**Comment by dsyme** [comment]
Yes, please implement IDisposable on IFsiEvaluator, thanks!  It helps with running resource-hungry scripts for documentation generation in automated testing.
